src/deluge/memory/general_memory_allocator.cpp

```diff
@@ -56,10 +56,11 @@ void GeneralMemoryAllocator::checkStack(char const* caller) {
 
 	char a;
 
-	int32_t distance = (int32_t)&a - (kInternalMemoryEnd - kProgramStackMaxSize);
+	int32_t distance = (int32_t)&a - (uint32_t)&program_stack_start;
```

Comment:
if this is the last use of kProgramStackMaxSize, could you also remove it from the definitions so we don't have a misleading constant around?

Reply:
Yeah good call